Week-4/promise.js

```diff
@@ -0,0 +1,36 @@
+const num = 5;
+// function that generates random numbers divisible by n with a default
+//upper limit of 1000000
+const specialRandom = (num = 1, limit = 1000000) => {
+   // getting a random number
+   const random = Math.random() * limit;
+   // rounding it off to be divisible by num
+   const res = Math.round( random / num ) * num;
+   return res;
+};
+
+const myPromise = () => {
+    return new Promise((resolve, reject) => {
```

Comment:
This needs a custom promise.Pleas watch and re-look into this question : https://www.youtube.com/watch?v=1l4wHWQCCIc

Reply:
sure